packages/url/src/index.ts

```diff
@@ -35,12 +33,8 @@ export class Url extends Connector {
   private async isomorphicInitialize() {
```

Comment:
network returned the provider to be set as part of activate, and it felt cleaner (mostly because error propagation could be handled in the activate method).